### PR TITLE
Include pointer tagging and cpython performance papers

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2129,3 +2129,23 @@ @article{dijkstra1978gc
     year = {1978},
     pages = {966--975},
 }
+
+@inproceedings{barany2014python,
+    author = {Barany, Gerg\"{o}},
+    title = {Python Interpreter Performance Deconstructed},
+    booktitle = {Dynamic Languages and Applications},
```
Comment (ltratt) [inline softdev.bib]: `booktitle = {DYLA}` (and remove `series`).

File: softdev.bib [hunk 2]
```diff
@@ -2129,3 +2129,23 @@ @article{dijkstra1978gc
...
+    series = {Dyla'14},
+    year = 2014,
+    month = Jun,
+    pages = {5:1--5:9},
+    publisher = {ACM},
+}
```
Comment (ltratt) [inline softdev.bib]: What does `5:` refer to?

File: softdev.bib [hunk 3]
```diff
@@ -2129,3 +2129,23 @@ @article{dijkstra1978gc
...
+
+@book{gudeman1995representing,
+    title={Representing type information in dynamically typed languages},
```
Comment (ltratt) [inline softdev.bib]: Missing spaces around `=`.

File: softdev.bib (hunk 2 above)
Comment (jacob-hughes) [reply softdev.bib]: I'm not sure. Truth be told I copied this from the bibtex citation notes in google scholar.

File: softdev.bib (hunk 1 above)
Comment (jacob-hughes) [reply softdev.bib]: Ok done.

File: softdev.bib (hunk 3 above)
Comment (jacob-hughes) [reply softdev.bib]: Fixed

File: softdev.bib (hunk 2 above)
Comment (ltratt) [reply softdev.bib]: I suggest we remove the `pages` key then.

Comment (jacob-hughes) [reply softdev.bib]: Sure

**Comment by jacob-hughes** [comment]
Pushed review fixes.

**Comment by ltratt** [comment]
Before I ask you to squash, do either of these have DOIs? It's nice, when possible, to add a `doi` field (but not everything has a DOI).

**Pull request merged**